doc: update docs for 2.9.2 (#22215)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-2.9.2.rst

See the :ref:`installation` section for more information about supported operating systems and toolchain.

Supported modem firmware
************************

See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.

Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
See :ref:`nrf9160_gs_updating_fw_modem` for instructions.

Modem-related libraries and versions
====================================

.. list-table:: Modem-related libraries and versions
   :widths: 15 10
   :header-rows: 1

   * - Library name
     - Version information
   * - Modem library
     - `Changelog <Modem library changelog for v2.9.2_>`_
   * - LwM2M carrier library
     - `Changelog <LwM2M carrier library changelog for v2.9.2_>`_

Known issues
************

Known issues are only tracked for the latest official release.
See `known issues for nRF Connect SDK v2.9.2`_ for the list of issues valid for the latest release.

.. _ncs_release_notes_292_changelog:

Changelog
*********

> 

Reviewer: @divipillai 
I would like to add following changelog entry for the following sdk-zephyr PR: https://github.com/nrfconnect/sdk-zephyr/pull/2836
```
Drivers
=====

nrfx
----

* Changed:

  * Updated MDK to version 8.68.2 to have improved errata handling for nRF54L05 and nRF54L10 devices.
```

is `Drivers` section correct here? this is `sdk-zephyr`-related rather than `sdk-nrf`-related

Author: We normally have a dedicated section for nrfx.  For example - https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.5.1.html#nrfx. 
Added the entry.

The following sections provide detailed lists of changes by component.

Build and configuration system
==============================

* Use the nrfutil-device v2.8.8 and SEGGER J-Link |jlink_ver| for the |NCS| v2.9.2 release.

Protocols
=========

This section provides detailed lists of changes by :ref:`protocol <protocols>`.

Matter
------

* Added:

  * Automatic Wi-Fi reconnection after RPU recovery.
  * Fixes for CVE-2024-56318 and CVE-2024-56319.

* Fixed:

>   * Problems with certification tests TC-DGGEN-2.1 and TC-IDM-10.5.

Reviewer 2: There was one more fix:
```suggestion
  * Problems with certification tests TC-DGGEN-2.1, TC-IDM-10.5 and TC-ACE-2.2.
```

Author: Updated.

Supported modem firmware
************************

See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.

Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
See :ref:`nrf9160_gs_updating_fw_modem` for instructions.

Modem-related libraries and versions
====================================

.. list-table:: Modem-related libraries and versions
   :widths: 15 10
   :header-rows: 1

   * - Library name
     - Version information
   * - Modem library
     - `Changelog <Modem library changelog for v2.9.2_>`_
   * - LwM2M carrier library
     - `Changelog <LwM2M carrier library changelog for v2.9.2_>`_

Known issues
************

Known issues are only tracked for the latest official release.
See `known issues for nRF Connect SDK v2.9.2`_ for the list of issues valid for the latest release.

.. _ncs_release_notes_292_changelog:

Changelog
*********

The following sections provide detailed lists of changes by component.

Build and configuration system
==============================

> * Use the nrfutil-device v2.8.8 and SEGGER J-Link |jlink_ver| for the |NCS| v2.9.2 release.

Author: @shanthanordic - do we need to explicitly mention in the release note to use nrfutil-device v2.8.8 and J-link v8.18 since earlier versions were v2.7.2 and v7.94i respectively for NCS 2.9.x releases.

Protocols
=========

This section provides detailed lists of changes by :ref:`protocol <protocols>`.

Matter
------

* Added:

>   * Automatic Wi-Fi reconnection after RPU recovery.

Reviewer 3: ```suggestion
  * Automatic Wi-Fi® reconnection after RPU recovery.
```

  * Fixes for CVE-2024-56318 and CVE-2024-56319.

* Fixed:

  * Problems with certification tests TC-DGGEN-2.1, TC-IDM-10.5, and TC-ACE-2.2.
  * Door lock issues related to credentials persistency and factory reset.

Matter fork
+++++++++++

* Added a new ``kFactoryReset`` event that is posted during factory reset.
  The application can register a handler and perform additional cleanup.

Zigbee
------

* Fixed compilation errors that previously occurred when the :kconfig:option:`CONFIG_ZIGBEE_FACTORY_RESET` Kconfig option was disabled.

Applications
============

This section provides detailed lists of changes by :ref:`application <applications>`.

Thingy:53: Zigbee weather station
---------------------------------

* Added:

  * A fix for logging negative temperature values.
  * Logging unification.

Libraries
=========

This section provides detailed lists of changes by :ref:`library <libraries>`.

Libraries for networking
------------------------

* :ref:`lib_nrf_provisioning` library:

  * Fixed an issue where the results from the :c:func:`zsock_getaddrinfo` function were not freed when the CoAP protocol was used for connection establishment.

nrfx
====

> * Updated MDK to v8.68.2 to have improved errata handling for nRF54L05 and nRF54L10 devices.

Reviewer 4: ```suggestion
* Updated MDK to v8.68.2 to improve errata handling for nRF54L05 and nRF54L10 devices.
```

Zephyr
======

> The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``beb733919d8d64a778a11bd5e7d5cbe5ae27b8ee``.

Reviewer 5: it's correct

.. _ncs_release_notes_292:

|NCS| v2.9.2 Release Notes
##########################

.. contents::
   :local:
   :depth: 2

|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF54, nRF70, and nRF91 Series.
The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.

Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
To learn more, see :ref:`software_maturity`.

Highlights
**********

This patch release adds changes on top of the :ref:`nRF Connect SDK v2.9.0 <ncs_release_notes_290>` and :ref:`nRF Connect SDK v2.9.1 <ncs_release_notes_291>`.

> The changes affect Matter, Zigbee, and nrfx.

Reviewer 6: Yes for NCs 2.9.2 it needs to be stated explicitly
